publish/publish.proj

```diff
@@ -11,7 +11,7 @@
   
   <Target Name="PublishFinalOutput"
           Condition="'$(Finalize)' == 'true'"
-          DependsOnTargets="FinalizeBuildInAzure;PublishCoreHostPackagesToFeed;UpdatePublishedVersions" />
+          DependsOnTargets="FinalizeBuildInAzure;PublishCoreHostPackagesToFeed;PublishDebToolPackagesToFeed;UpdatePublishedVersions" />
```

Comment:
I don't think `PublishFinalOutput` should be dependent on `PublishDebToolPackagesToFeed` anymore